functions/package.json

```diff
@@ -26,10 +26,10 @@
     "firebase-functions": "^3.13.2"
   },
   "devDependencies": {
-    "typescript": "^4.2.3",
+    "@types/node": "^14.14.37",
```

Comment:
nit: remove these changes

Reply:
This is more alphabetical, so i'mma keep it, also I didn't do this so I'm not sure how this happened.